pkg/tracker/currentVariables.go

```diff
@@ -49,44 +54,37 @@ func (b *CurrentVariablesTracker) Exec(ctx context.Context) error {
 	// if we've mined it, don't save it.
 	myStatus, err := instance.DidMine(nil, currentChallenge, fromAddress)
 	if err != nil {
-		fmt.Println("My Status Retrieval Error")
-		return err
+		return errors.Wrap(err, "Status Retrieval")
 	}
 	bitSetVar := []byte{0}
 	if myStatus {
 		bitSetVar = []byte{1}
 	}
-	currentVarsLog.Info("Retrieved variables. challengeHash: %x", currentChallenge)
+	level.Info(b.logger).Log("msg", "Retrieved variables", "challengeHash", currentChallenge)
```

Comment:
lets be consistent with all lower case for errors and logs.
```suggestion
	level.Info(b.logger).Log("msg", "retrieved variables", "challengeHash", currentChallenge)
```